doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -97,8 +97,12 @@ All this process happens per each NGSIEvent aggregated.
 
```

Comment:
Should last_data_function.md linked from mysql sink documentation (as Postgresql/PostGIS sink doc I think do)?

Reply:
Fixed in [3a6ef257](https://github.com/telefonicaid/fiware-cygnus/pull/1944/commits/3a6ef257335bcb2c419b38e841d7a683dfeabe76)

Reply:
What about a link also in doc/cygnus-ngsi/flume_extensions_catalogue/README.md ?

Reply:
My mistake.

Fixed in [e1fc9bf](https://github.com/telefonicaid/fiware-cygnus/pull/1944/commits/e1fc9bf8fa00baec03b7aaba267b440468a6ce71).